tests/Behat/Page/Order/IndexPage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\CustomerReorderPlugin\Behat\Page\Order;
+
+use Behat\Mink\Element\NodeElement;
+use \Sylius\Behat\Page\Shop\Account\Order\IndexPage as BaseIndexPage;
+
+final class IndexPage extends BaseIndexPage implements IndexPageInterface
+{
+    public function clickReorderButtonNextToTheOrder(string $orderNumber): void
+    {
+        $orderData = $this->getOrderData($orderNumber);
+
+        $reorderButton = $orderData->find('css', 'td:last-child')->find('css', 'button');
```

Comment:
This reverts the solution I've implemented in #30 that prevents Behat failing when run together with Sylius/CustomerOrderCancellationPlugin.

Reply:
Sorry! Fixed.